src/SFA.DAS.AssessorService.Web.Staff/Views/Apply/Financial/Graded.cshtml

```diff
@@ -15,7 +15,7 @@
                 <span class="bold">@Model.QnAData.FinancialApplicationGrade.SelectedGrade</span>.
             </p>
         </div>
-        <p>You can view previous evaluations, including this one in the <a asp-action="PreviousApplications" asp-controller="Financial">previous assessments</a> area.</p>
+        <p>You can view previous evaluations, including this one in the <a asp-action="ClosedApplications" asp-controller="Financial">completed assessments</a> area.</p>
```

Comment:
Why is the action method not called CompletedApplications?

Reply:
This has been changed